### PR TITLE
[TOC][Put VSTO in a top level node]

### PR DESCRIPTION
**Pull request by mscharlock**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b667ed9](https://github.com/OfficeDev/office-js-docs-pr/commits/b667ed90f293361b3ef8c6eb78751882dfbe9ed1)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-2801f5d2)

<a id="2801f5d2"></a>
###
 - **[Error-[PullRequestMergeConflict](https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/)]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008171946456398-2096%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=DjPk%2F1iQ3a0kIUuzzltE1Enpc3P%2FukRtx41ZP0osxcE%3D&st=2020-08-17T19%3A42%3A03Z&se=2020-09-17T19%3A47%3A03Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://teams.microsoft.com/l/message/19:7ecffca1166a4a3986fed528cf0870ee@thread.skype/1590030602688?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&parentMessageId=1590030602688&teamName=Docs%20Support&channelName=General&createdTime=1590030602688) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3e670c4](https://github.com/OfficeDev/office-js-docs-pr/commits/3e670c417d5d22ac86b27b21aea10e9f837d2320)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/concepts/browsers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/browsers-used-by-office-web-add-ins?branch=pr-en-us-2096) |
[docs/concepts/requirements-for-running-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/concepts/requirements-for-running-office-add-ins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/requirements-for-running-office-add-ins?branch=pr-en-us-2096) |
[docs/develop/create-sso-office-add-ins-aspnet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/develop/create-sso-office-add-ins-aspnet.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/create-sso-office-add-ins-aspnet?branch=pr-en-us-2096) |
[docs/develop/create-sso-office-add-ins-nodejs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/develop/create-sso-office-add-ins-nodejs.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/create-sso-office-add-ins-nodejs?branch=pr-en-us-2096) |
[docs/excel/configure-your-add-in-to-use-a-shared-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/excel/configure-your-add-in-to-use-a-shared-runtime.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/configure-your-add-in-to-use-a-shared-runtime?branch=pr-en-us-2096) |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-overview?branch=pr-en-us-2096) |
[docs/excel/custom-functions-shared-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/excel/custom-functions-shared-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-shared-overview?branch=pr-en-us-2096) |
[docs/excel/make-custom-functions-compatible-with-xll-udf.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/excel/make-custom-functions-compatible-with-xll-udf.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/make-custom-functions-compatible-with-xll-udf?branch=pr-en-us-2096) |
[docs/includes/shared-runtime-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/includes/shared-runtime-note.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-authentication?branch=pr-en-us-2096) |
[docs/outlook/append-on-send.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/outlook/append-on-send.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/append-on-send?branch=pr-en-us-2096) |
[docs/outlook/autolaunch.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/outlook/autolaunch.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/autolaunch?branch=pr-en-us-2096) |
[docs/quickstarts/outlook-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/quickstarts/outlook-quickstart.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/outlook-quickstart?branch=pr-en-us-2096) |
[docs/reference/javascript-api-for-office-error-codes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/reference/javascript-api-for-office-error-codes.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/javascript-api-for-office-error-codes?branch=pr-en-us-2096) |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/browsers-used-by-office-web-add-ins?branch=pr-en-us-2096) |
[docs/tutorials/outlook-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/tutorials/outlook-tutorial.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/tutorials/outlook-tutorial?branch=pr-en-us-2096) |
[docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial?branch=pr-en-us-2096) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008171949125620-2096%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=x50qa6g0EUUvHBC97I8y83UMmIFhEapX0p2YJ7AQI24%3D&st=2020-08-17T19%3A44%3A49Z&se=2020-09-17T19%3A49%3A49Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/toc.yml [hunk 1]
```diff
@@ -1156,12 +1153,19 @@
...
   - name: Handle high DPI and DPI scaling in your Office solution
     href: /office/client-developer/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution
+
+- name: VSTO transition resources
+  items: 
+    - name: VSTO add-in developers' guide
+      href: overview/learning-path-transition.md
+    - name: Share code from VSTO Add-in to Office add-in
+      href: tutorials/migrate-vsto-to-office-add-in-shared-code-library-tutorial.md
+    - name: Identify an equivalent COM add-in
```
Comment (AlexJerabek) [inline docs/toc.yml:1170]: Did this "Identify an equivalent COM add-in" get moved from somewhere in the TOC?

Comment (mscharlock) [reply docs/toc.yml:1170]: Yep, came from some other node that dealt with generic concepts or something

Comment (ElizabethSamuel-MSFT) [reply docs/toc.yml:1170]: So should there be a deleted entry for it in the TOC file?

File: docs/toc.yml [hunk 2]
```diff
@@ -1156,12 +1153,19 @@
...
-    href: tutorials/migrate-vsto-to-office-add-in-shared-code-library-tutorial.md
   - name: Batching custom function remote calls
     href: excel/custom-functions-batching.md
     displayName: Excel, Custom Functions
   - name: Authentication
     href: develop/authentication-pnp.md
   - name: Handle high DPI and DPI scaling in your Office solution
     href: /office/client-developer/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution
+
+- name: VSTO transition resources
```
Comment (ElizabethSamuel-MSFT) [inline docs/toc.yml:1164]: If this includes COM resources, should it say so? e.g.:
```suggestion
- name: VSTO and COM transition resources
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[abc73c2](https://github.com/OfficeDev/office-js-docs-pr/commits/abc73c2d6e64ee5b1f2236bacd4cf43a0ad91653)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/add-in-development-best-practices?branch=pr-en-us-2096) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C19%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008192130467290-2096%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=8qjYLxY45c0Kj0p1M6QQcg64o68q7zKkYw7DPMpKVJ8%3D&st=2020-08-19T21%3A26%3A27Z&se=2020-09-19T21%3A31%3A27Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[dd85c43](https://github.com/OfficeDev/office-js-docs-pr/commits/dd85c43864b7e76ad96f3ba4811887ad38f4df19)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/add-in-development-best-practices?branch=pr-en-us-2096) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008261905161262-2096%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=xMZtDNxVTICzeb%2FmdUsQSBLZpDP9rKy%2Fqt5vFy8aIo8%3D&st=2020-08-26T19%3A01%3A04Z&se=2020-09-26T19%3A06%3A04Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[745df8d](https://github.com/OfficeDev/office-js-docs-pr/commits/745df8dbbca4835f3abf0adf0d08aa02365cc7de)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-tocvsto/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/add-in-development-best-practices?branch=pr-en-us-2096) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008261906537584-2096%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ZqTkzilPz8%2FAOuvH1qeGSPGeR9d0DbJQMa3EU5vZEzo%3D&st=2020-08-26T19%3A02%3A31Z&se=2020-09-26T19%3A07%3A31Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**